docs/source/model-registry.rst

```diff
@@ -16,31 +16,31 @@ staging to production), and annotations.
 Concepts
 ========
 
-The MLflow Model Registry is a standalone component that introduces several concepts that
-describe the full lifecycle of an MLflow Model.
+As the fourth standalone component of MLflow, the Model Registry introduces few concepts that describe and facilitate the full lifecycle of an MLflow model.
 
 Model
-    An MLflow Model logged with one of the model flavor’s ``log_model`` methods.
+    An MLflow Model is created from an experiment or run that is logged with one of the model flavor’s ``mlflow.model_flavor.log_model`` methods. Once logged, this model can then be registered with the Model Registry.
 
 Registered Model
-    An MLflow Model registered with the MLflow Model Registry. A registered model has a unique name, contains versions, and other metadata.
+    An MLflow Model can be registered with the  Model Registry. It has a unique name, contains versions, associated transitional stages, model lineage, and other metadata.
```

Comment:
```suggestion
    An MLflow Model can be registered with the  Model Registry. A registered model has a unique name, contains versions, associated transitional stages, model lineage, and other metadata.
```

Reply:
Be careful about replacing nouns with pronouns. In your previous version, it's not clear if "It" refers to the Model Registry or the Model.